ERCS/erc-7776.md

```diff
@@ -0,0 +1,554 @@
+---
+eip: 7776
+title: Transparent Financial Statements
+description: A.P.I. for Protocols to Ensure Open Transparent Financial Data for all Investors.
+author: Ignacio Ceaglio (@Nachoxt17) <ignacioceaglio@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-xxxx-transparent-financial-statements/21191
+status: Review
+type: Standards Track
+category: ERC
+created: 2024-09-20
+requires: 20
+---
+
+## Abstract
+
+This proposal defines a standard A.P.I. that enables E.V.M. Blockchain-based companies (or also called "Protocols") to publish their financial information, specifically Income Statements and Balance Sheets, on-chain in a transparent and accessible manner through Solidity Smart Contracts. This standard aims to emulate the reporting structure used by publicly traded companies in traditional stocks markets, like the S.E.C. 10-Q filings. The financial statements include key information, namely as Revenue, Cost of Goods Sold, Operating Expenses, Operating Income, Earnings before Interest, Taxes, Depreciation, and Amortization (E.B.I.T.D.A.) and
+Earnings Per Token (E.P.Share-Token), allowing investors to assess the financial health of blockchain-based companies in a standardized, transparent, clear and reliable format.
+
+## Motivation
+
+The motivation of this E.I.P. is to Bring Seriousness to the CryptoCurrencies Investments Market. Currently, the situation is as follows:
+
+Most [ERC-20](./eip-20.md) Tokens representing E.V.M. Blockchain-based companies (or also called "Protocols"), DO NOT work the same way as a Publicly Traded Stock that represents a Share of ownership of the equity of that such company (so the user who buys a Protocol's ERC-20, is also now a share-holder and co-owner of the business, its profits and/or its dividends), but rather function as "Commodities" such as oil; they are consumable items created by said E.V.M. Blockchain-based company (or "Protocol") to be spent in their platform. They are Publicly Traded and advertised to be representing the underlying Protocol like a Share, working in practice the same way as a Commodity and without any Public, Transparent and _Clear_ Financial Information as publicly traded stocks have.
+
+Added to that, most token research analysis reports that can be currently found on the internet are informal Substack or Twitter posts, with lots of abstract explanations about the features of the said Protocol to invest in, that lack of transparent financial numbers and factual financial information, that are made by anonymous users without real exposed reputations to affect.
+
+This E.I.P. will improve that by giving users and investors transparent, clear and factual financial information to work with when analyzing as a potential investment the such
```

Comment:
I haven't read the specification section yet, but I suspect "factual" here doesn't mean "enforced by logic running on the blockchain." I'd be concerned you're overstating what this proposal can accomplish.

Reply:
Good point. In brief, the information is provided and upgraded by the same company/protocol that is using their own implementation of ERC-7776, directly from their business smart contracts (directly from their blockchain “bank accounts”).

Each protocol has different smart contracts and ways to generate revenue, name methods, they handle different assets, etc. So the Standard is made in a way to make that adjustable from different cases.

Why would they do that at all? Easy; it’s in their own interest to provide this information to generate trust among investors, and thus to potentially receive more investments.

I just added this line to the specification:

> If the contract owner uses data or methods from other owned smart contracts external to their smart contract implementation of this standard, those smart contracts MUST be verified in the correspondent blockchain explorer and of open and visible source code.

@SamWilsn  Do you think that there is any better way to enforce or explain this? The blockchain data already exists, my objective is that it is displayed in an open, clear and standardized way directly from the smart contracts of the protocol.